lib/main.dart

```diff
@@ -1,65 +1,29 @@
 import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:qlink/theme.dart';
+import 'package:qlink/screens/screens.dart';
 
 void main() {
-  runApp(MyApp());
+  _setSystemColors();
```

Comment:
kinda weird to have this in `main`. is this something you saw suggested on a web article?

otherwise I'd think color definitions belong with the primary app widget

Reply:
These are native system colors, so they need to be set in `main`, pretty sure